coursefinder/templates/coursefinder/partials/filters.html

```diff
@@ -29,8 +15,10 @@
 
                 <input id="page_input" name="page" type="text" value="{{search.page}}" hidden />
                 <input id="count_input" name="count" type="text" value="{{search.count}}" hidden />
+                <input id="sort_by_subject" name="sort_by_subject" type="text" value="{{search.sortBySubject}}" hidden />
 
                 <input name="course_query" id="course_query" type="text" value="{{filter_form.course_query}}" hidden />
+                <input name="institution_query" id="institutions_query" type="text" {% if filter_form.institutions %} value='"{{filter_form.institutions}}"' {% endif %} hidden />
```

Comment:
Did you add this hidden input for the institution query? I removed it after I added the institutions dropdown as it was no longer needed?

Reply:
Ah - that was caused by the rebase - I'll remove it if you don't need it